python/tchannel/handler.py

```diff
@@ -94,21 +99,29 @@ class TChannelResponse(object):
     """TChannel Response Wrapper"""
 
     __slots__ = ('_connection', '_request',
-                 'resp_msg', 'id')
+                 'resp_msg', 'id', 'body',
+                 'headers')
```

Comment:
Something's wrong here, we already have a `message` here, so any `headers`/`body` we write should be written to `message.arg_2` and `message.arg_3`, right?